requirements.txt

```diff
@@ -1,2 +1,4 @@
 django>=1.11
+# https://stackoverflow.com/questions/68024060/assertionerror-database-connection-isnt-set-to-utc
+psycopg2==2.8.6
```

Comment:
It can be an issue when we want upgraded version of psycopg2 for Django3
Maybe use it only for test? 🤔

Reply:
Yep